setup.py

```diff
@@ -10,7 +10,7 @@ def read(*parts):
 
 
 setup(
-    name="django-dbtemplates",
+    name="django-dbtemplates-iplweb",
```

Comment:
What's blocking releasing under the main name?

```suggestion
    name="django-dbtemplates",
```